CRM/Core/Region.php

```diff
@@ -56,6 +56,7 @@ public function __construct($name) {
    * @return string, HTML
    */
   public function render($default, $allowCmsOverride = TRUE) {
+    $allowCmsOverride = FALSE;
```

Comment:
@totten isn't that defeating the purposes of the param being passed into the function? I'm guessing this is testing

Reply:
Yeah, that was the idea. :) But it's the "(REVERT)" commit.